core/libs/commonwealth/commonwealth/mavlink_comm/MavlinkComm.py

```diff
@@ -21,6 +25,7 @@ def __init__(self) -> None:
         self.m2r_address = "localhost:6040"
 
     def set_system_id(self, system_id: int) -> None:
+        logger.info(f"system_id detected as: {system_id}")
```

Comment:
```suggestion
        logger.info(f"system_id set to: {system_id}")
```
This is a setter function - there's no obvious context in this function as to why the value being set has been "detected" rather than just set as a user preference.

Reply:
very good point, the detected thing should be where it is actually detected.